source/main.c

```diff
@@ -179,7 +182,11 @@ extern u8 __xfb[];
 
 int main()
 {
+    // If this is set to 1 force the
```

Comment:
Force the what?

Reply:
Good question. This is from seewood's commit, but you are right that this `forceIPL` is never used. Will remove.